2-TokenCache/README.md

```diff
@@ -174,6 +174,16 @@ cacheHelper.RegisterCache(app.UserTokenCache);
    - `GetAccountsAsync` will return all account in the cache
    - `RemoveAsync` will remove the account from the cache
 
+### Using Embedded View - supported on **.NET 4.2.7 only**
```

Comment:
I would say **.NET framework** only (no specifically the version)

Reply:
text changed